lib/js/index.js

```diff
@@ -291,17 +331,30 @@ class DateTimePicker extends Events {
   setTime(time) {
     const m = moment(time);
     const hour = m.format('HH');
+    const minutes = m.format('mm');
     const hourAsInt = parseInt(hour, 10) % 12;
+    const minuteAsInt = parseInt(minutes, 10);
     const oldActiveHours = this.$(`.c-datepicker__clock__hours .${this.options.styles.clockNum}--active`);
```

Comment:
`.js-clock-hours`, please - this one was here already but this seems like a good time to improve the code quality while making similar changes 😄